### ISSUE TITLE
errors with 'vertex coincides with origin'

### ISSUE THREAD

**Issue by KyleLee1999** [mdickinson/polyhedron#8]
I don't really get what does this mean? Could anyone help me?

**Comment by mdickinson** [comment]
You'll get this error if you try to compute the winding number of a polyhedron around a point that is itself a vertex of that polyhedron (or similarly for polygons) - in that case, the winding number is not defined.

Does that help? If not, please give more information on exactly what you're doing, what results you get, and what results you expected.

**Comment by KyleLee1999** [comment]
yes. You are right. I am trying to remove those suspicious data. thanks again.

Sent from my iPhone

On 22 Sep 2022, at 17:13, Mark Dickinson ***@***.***> wrote:

﻿

You'll get this error if you try to compute the winding number of a polyhedron around a point that is itself a vertex of that polyhedron (or similarly for polygons) - in that case, the winding number is not defined.

Does that help? If not, please give more information on exactly what you're doing, what results you get, and what results you expected.
